test/e2e/pull_test.go

```diff
@@ -84,6 +84,35 @@ var _ = Describe("Podman pull", func() {
 		Expect(session).Should(Exit(0))
 	})
 
+	It("podman pull and run on split imagestore", func() {
+		SkipIfRemote("podman-remote does not support setting external imagestore")
+
+		tmpDir := filepath.Join(podmanTest.TempDir, "splitstore")
```

Comment:
Can you setup a STORAGE_CONF=tmpDir/storage.conf file to make sure that setting this in storage.conf actually works.

Reply:
Yes added `storage.conf` below.